docs/melange-book/index.md

```diff
@@ -0,0 +1,16 @@
+# Melange Book
+
+## Motivation
+
+This is a project-based, guided introduction to Melange and its ecosystem.
+Because Melange integrates both OCaml and JavaScript ecosystems, there are a lot
+of tools and topics to learn, which can be overwhelming for newcomers. Therefore
+we try to make each chapter small and digestible, not introducing too many
+things at once.
+
+## Audience
+
+You should already know how to make frontend applications in JavaScript, and
+familiarity with React will be very helpful. You do not need to know
+OCaml/Reason, we'll slowly introduce the basics of the language throughout the
```

Comment:
I'd just write OCaml.

Reply:
Will just avoid writing "Reason", unless it's part of "ReasonReact"